front/src/applications/operationalStudies/views/SimulationResults.tsx

```diff
@@ -198,13 +195,30 @@ const SimulationResults = ({
         </div>
       )}
 
+      {/* SIMULATION : MAP */}
+      <div className="simulation-map">
+        <SimulationResultsMap
+          setExtViewport={setExtViewport}
+          geometry={pathProperties?.geometry}
+          trainSimulation={
+            selectedTrainSchedule && trainSimulation
+              ? {
+                  ...trainSimulation,
+                  trainId: selectedTrainSchedule.id,
+                  startTime: selectedTrainSchedule.start_time,
+                }
+              : undefined
+          }
+        />
+      </div>
+
       {/* TIME STOPS TABLE */}
       {selectedTrainSchedule &&
         trainSimulation.status === 'success' &&
         pathProperties &&
         operationalPoints &&
         infraId && (
-          <div className="osrd-simulation-container mb-2">
+          <div className="osrd-simulation-container mb-4">
```

Comment:
Margin below the output table is like this : 

![image](https://github.com/user-attachments/assets/31c0772c-ef89-4d8b-982a-d0985e5f2e41)

Reply:
I think it's done :white_check_mark: